src/index.css

```diff
@@ -15,6 +15,9 @@
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+  --primary: #8772FF;
```

Comment:
add it in tailwind also, look at how we add background and foreground in  tailwind

Reply:
updated